cranelift/filetests/filetests/isa/x64/load-op.clif

```diff
@@ -0,0 +1,43 @@
+test compile
+target x86_64
+feature "experimental_x64"
+
+function %add_from_mem_u32_1(i64, i32) -> i32 {
+block0(v0: i64, v1: i32):
+  v2 = load.i32 v0
+  v3 = iadd.i32 v2, v1
+  ; check: addl    0(%rdi),
```

Comment:
Confusing that the second operand of the `addl` is missing; can this be added?  Ditto for the cases below.

Reply:
The idea here was to make the test invariant to possible regalloc changes; for some reason it's using `r12` here, which seemed odd to me. But, I suppose it's clearer to just do the Simple Thing and match the whole instruction; fixed.